### ISSUE TITLE
Hangs on Connecting to rotki backend

### ISSUE THREAD

**Issue by timnugent** [rotki/rotki#7983]
Rotki UI fails to load with the message "Connecting to rotki backend". Version: 0.1.29 (v1.33.0 upstream) on Dappnode. Restarting fails to resolve this. Logs follow:

2024-05-26T17:28:56.788965466Z INFO:monitor:Preparing to cleanup tmp directory
2024-05-26T17:28:56.789202082Z INFO:monitor:Deleted 0 files or directories, skipped 0 from /tmp
2024-05-26T17:28:56.789465718Z INFO:monitor:no config file provided
2024-05-26T17:28:56.789484541Z INFO:monitor:loading config from env
2024-05-26T17:28:56.789517019Z INFO:monitor:starting rotki backend
2024-05-26T17:28:56.789663734Z INFO:monitor:starting nginx
2024-05-26T17:28:56.790236625Z 42
2024-05-26T17:28:56.794092715Z 2024/05/26 17:28:56 [notice] 10#10: using the "epoll" event method
2024-05-26T17:28:56.794107052Z 2024/05/26 17:28:56 [notice] 10#10: nginx/1.25.5
2024-05-26T17:28:56.794109837Z 2024/05/26 17:28:56 [notice] 10#10: built by gcc 12.2.0 (Debian 12.2.0-14) 
2024-05-26T17:28:56.794111789Z 2024/05/26 17:28:56 [notice] 10#10: OS: Linux 5.10.0-9-amd64
2024-05-26T17:28:56.794113533Z 2024/05/26 17:28:56 [notice] 10#10: getrlimit(RLIMIT_NOFILE): 1048576:1048576
2024-05-26T17:28:56.794177107Z 2024/05/26 17:28:56 [notice] 10#10: start worker processes
2024-05-26T17:28:56.794275601Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 11
2024-05-26T17:28:56.794363212Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 12
2024-05-26T17:28:56.794435689Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 13
2024-05-26T17:28:56.794511008Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 14
2024-05-26T17:28:56.794605655Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 15
2024-05-26T17:28:56.794708832Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 16
2024-05-26T17:28:56.794822843Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 17
2024-05-26T17:28:56.794915203Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 18
2024-05-26T17:28:56.795015526Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 19
2024-05-26T17:28:56.795114517Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 20
2024-05-26T17:28:56.795250936Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 21
2024-05-26T17:28:56.795342109Z 2024/05/26 17:28:56 [notice] 10#10: start worker process 22
2024-05-26T17:28:57.452737050Z 2024/05/26 17:28:57 [error] 11#11: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://127.0.0.1:4242/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.452785164Z 2024/05/26 17:28:57 [warn] 11#11: *1 upstream server temporarily disabled while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://127.0.0.1:4242/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.452790517Z 2024/05/26 17:28:57 [error] 11#11: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://127.0.0.1:4242/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.452800181Z 2024/05/26 17:28:57 [warn] 11#11: *1 upstream server temporarily disabled while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://127.0.0.1:4242/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.452831750Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.481392520Z 2024/05/26 17:28:57 [error] 11#11: *1 no live upstreams while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://localhost/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.481436279Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.497422722Z 2024/05/26 17:28:57 [error] 11#11: *1 no live upstreams while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://localhost/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.497452324Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.515498052Z 2024/05/26 17:28:57 [error] 11#11: *1 no live upstreams while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://localhost/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.515621793Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.535615911Z 2024/05/26 17:28:57 [error] 11#11: *1 no live upstreams while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://localhost/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.535704089Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.557232542Z 2024/05/26 17:28:57 [error] 11#11: *1 no live upstreams while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://localhost/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.557281331Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.574389429Z 2024/05/26 17:28:57 [error] 11#11: *1 no live upstreams while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://localhost/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.574405838Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.594115903Z 2024/05/26 17:28:57 [error] 11#11: *1 no live upstreams while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://localhost/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.594189045Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.613846065Z 2024/05/26 17:28:57 [error] 11#11: *1 no live upstreams while connecting to upstream, client: 172.33.0.22, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://localhost/api/1/ping", host: "rotki.dappnode", referrer: "http://rotki.dappnode/"
2024-05-26T17:28:57.613890337Z 172.33.0.22 - - [26/May/2024:17:28:57 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "http://rotki.dappnode/" "Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Mobile Safari/537.36" "-"
2024-05-26T17:28:57.864144825Z eth_utils/network.py:44: UserWarning: Network 345 with name 'Yooldo Verse Mainnet' does not have a valid ChainId. eth-typing should be updated with the latest networks.
2024-05-26T17:28:57.864869149Z eth_utils/network.py:44: UserWarning: Network 12611 with name 'Astar zkEVM' does not have a valid ChainId. eth-typing should be updated with the latest networks.
2024-05-26T17:28:59.309833323Z Traceback (most recent call last):
2024-05-26T17:28:59.309850552Z   File "rotkehlchen-script.py", line 2, in <module>
2024-05-26T17:28:59.309866032Z   File "rotkehlchen/__main__.py", line 37, in main
2024-05-26T17:28:59.309901130Z   File "rotkehlchen/server.py", line 63, in main
2024-05-26T17:28:59.309926040Z   File "rotkehlchen/api/server.py", line 494, in start
2024-05-26T17:28:59.309946564Z   File "gevent/baseserver.py", line 336, in start
2024-05-26T17:28:59.309977464Z   File "gevent/pywsgi.py", line 1669, in init_socket
2024-05-26T17:28:59.310002565Z   File "gevent/pywsgi.py", line 1681, in update_environ
2024-05-26T17:28:59.310022822Z   File "gevent/_socketcommon.py", line 286, in getfqdn
2024-05-26T17:28:59.310053017Z   File "gevent/_socketcommon.py", line 252, in gethostbyaddr
2024-05-26T17:28:59.310071743Z   File "gevent/resolver/thread.py", line 66, in gethostbyaddr
2024-05-26T17:28:59.310097242Z   File "gevent/pool.py", line 161, in apply
2024-05-26T17:28:59.310124379Z   File "gevent/threadpool.py", line 546, in spawn
2024-05-26T17:28:59.310161449Z   File "gevent/threadpool.py", line 492, in adjust
2024-05-26T17:28:59.310181813Z   File "gevent/threadpool.py", line 472, in _adjust_step
2024-05-26T17:28:59.310202129Z   File "gevent/threadpool.py", line 499, in _add_thread
2024-05-26T17:28:59.310222285Z   File "gevent/threadpool.py", line 106, in __init__
2024-05-26T17:28:59.310244592Z RuntimeError: can't start new thread
2024-05-26T17:28:59.310259076Z [23] Failed to execute script 'rotkehlchen-script' due to unhandled exception!
2024-05-26T17:28:59.310350114Z rotki REST API server is running at: 0.0.0.0:4242 with loglevel INFO
2024-05-26T17:29:26.828805778Z 2024/05/26 17:29:26 [error] 13#13: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://127.0.0.1:4242/api/1/ping", host: "localhost"

2024-05-26T17:29:26.828825619Z 2024/05/26 17:29:26 [warn] 13#13: *5 upstream server temporarily disabled while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://127.0.0.1:4242/api/1/ping", host: "localhost"
2024-05-26T17:29:26.828833095Z 2024/05/26 17:29:26 [error] 13#13: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://127.0.0.1:4242/api/1/ping", host: "localhost"

2024-05-26T17:29:26.828835922Z 2024/05/26 17:29:26 [warn] 13#13: *5 upstream server temporarily disabled while connecting to upstream, client: 127.0.0.1, server: _, request: "GET /api/1/ping HTTP/1.1", upstream: "http://127.0.0.1:4242/api/1/ping", host: "localhost"
2024-05-26T17:29:26.828890707Z 127.0.0.1 - - [26/May/2024:17:29:26 +0000] "GET /api/1/ping HTTP/1.1" 502 497 "-" "curl/7.88.1" "-"

**Comment by LefterisJP** [comment]
Hey @timnugent this is a known error and it's on the dappnode package side. There is some discussion in the rotki package there and in our and their discord.

TL;DR: Upgrade your docker version

our discord: https://discord.rotki.com/
dappnode discord: https://discord.gg/dappnode

**Comment by timnugent** [comment]
Thanks @LefterisJP - yes, looks like I can update Docker so will try that
